README.md

```diff
@@ -147,40 +156,76 @@ Scanner vendors are supposed to implement Scanner Adapters exposing the Scanner
 4. Wait 15 seconds or use your own retry interval ...
 5. ... and try getting scan report again:
    ```
-   curl -H 'Accept: application/vnd.scanner.adapter.vuln.report.harbor+json; version=1.0' \
+    curl -H 'application/vnd.security.vulnerability.report; version=1.1' \
      http://scanner-adapter:8080/api/v1/scan/3fa85f64-5717-4562-b3fc-2c963f66afa6/report
+    Content-Type: application/vnd.security.vulnerability.report; version=1.1;
+    Status: 200 OK
 
-   Content-Type: application/vnd.scanner.adapter.vuln.report.harbor+json; version=1.0
-   Status: 200 OK
-
-   {
-     "generated_at": "2019-08-07T12:17:21.854Z",
-     "artifact": {
-       "repository": "library/mongo",
-       "digest": "sha256:917f5b7f4bef1b35ee90f03033f33a81002511c1e0767fd44276d4bd9cd2fa8e"
-     },
-     "scanner": {
-       "name": "Microscanner",
-       "vendor": "Aqua Security",
-       "version": "3.0.5",
-     },
-     "severity": "High",
-     "vulnerabilities": [
-       {
-         "id": "CVE-2017-8283",
-         "package": "dpkg",
-         "version": "1.17.27",
-         "fix_version": "1.18.0",
-         "severity": "High",
-         "description": "...",
-         "links": [
-           "https://security-tracker.debian.org/tracker/CVE-2017-8283"
-         ]
-       },
-       ...
-     ]
-   }
-   ```
+    {
+        "generated_at": "2021-03-09T11:40:28.154072066Z",
+        "artifact": {
+          "repository": "library/docker",
+          "digest": "sha256:7215e8e09ea282e517aa350fc5380c1773c117b1867316fb59076d901e252d15",
+          "mime_type": "application/vnd.docker.distribution.manifest.v2+json"
+        },
+        "scanner": {
+          "name": "Trivy",
+          "vendor": "Aqua Security",
+          "version": "v0.16.0"
+        },
+        "severity": "High",
+      "vulnerabilities": [
```

Comment:
Indentation: I believe the `vulnerabilities` property is at the same level as `severity`.